src/components/frame/v5/pages/VerifiedPage/VerifiedPage.tsx

```diff
@@ -1,29 +1,19 @@
 import React, { FC } from 'react';
 
-import Navigation from '~v5/common/Navigation';
-import TwoColumns from '~v5/frame/TwoColumns';
-import Spinner from '~v5/shared/Spinner';
 import Table from './partials/Table';
 import { useVerifiedPage } from './hooks';
 
 const displayName = 'v5.pages.VerifiedPage';
 
 const VerifiedPage: FC = () => {
-  const { verifiedMembers, loadingMembers } = useVerifiedPage();
+  const { verifiedMembers } = useVerifiedPage();
 
   return (
-    <Spinner
-      loading={loadingMembers}
-      loadingText={{ id: 'loading.verifiedPage' }}
-    >
-      <TwoColumns aside={<Navigation pageName="members" />}>
-        <div className="flex justify-between mt-6">
-          <div className="w-full">
-            <Table list={verifiedMembers} onReputationSortClick={() => {}} />
-          </div>
-        </div>
-      </TwoColumns>
-    </Spinner>
+    <div className="flex justify-between mt-6">
+      <div className="w-full">
+        <Table list={verifiedMembers} onReputationSortClick={() => {}} />
+      </div>
```

Comment:
We can remove this div and apply the class to the parent `div` if it's still needed

Reply:
Just copied it. But will change.